File: P1S3/Formulario.cpp
//
// Created by ines on 8/4/21.
//

#include "Formulario.h"

Formulario(){}

Formulario::Formulario(unsigned e,string des,string d){
    edad = e;
    descripcion = des;
    prioridad = BAJA;
    dni = d;
}

void Formulario::setPrioridad(PRIORIDAD p) {
    prioridad = p;
}

unsigned Formulario::getEdad() {
    return edad;
}

void Formulario::setEdad(unsigned e){
    edad = e;
}

string Formulario::getDNI() {
    return dni;
}

void Formulario::setDNI(string d) {
    dni = d;
}

string Formulario::getDescripcion() {
    return descripcion;
}

void Formulario::setDescripcion(string d) {
    descripcion = d;
}

File: DSP1_S3_JAVA/src/Formulario.java
public class Formulario {
    int edad;
    String descripcion;
    Prioridad prioridad;
    String dni;
    boolean encriptado;

    public Formulario(){
        prioridad = Prioridad.BAJA;
        encriptado = false;
    }

    public Formulario(int e, String des, String d){
        edad = e;
        descripcion = des;
        dni = d;
        prioridad = Prioridad.BAJA;
        encriptado = false;

    }

    public int getEdad() {
        return edad;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public Prioridad getPrioridad() {
        return prioridad;
    }

    public String getDni() {
        return dni;
    }

    public boolean isEncriptado() {
        return encriptado;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public void setDni(String dni) {
        this.dni = dni;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public void setEncriptado(boolean encriptado) {
        this.encriptado = encriptado;
    }

    public void setPrioridad(Prioridad prioridad) {
        this.prioridad = prioridad;
    }

    public boolean getEncryptionStatus(){return encriptado;}
}
